### PR TITLE
Bump google-cloud-monitoring dependency version as required by #25.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Fixes the following error after #25:
```
Unable to export to Monitoring service because: unknown keyword: service_address
```

### CODE REVIEW

**Comment by igorpeshansky** [comment]
@davidbtucker FYI.

File: opencensus-stackdriver.gemspec [hunk 1]
```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "concurrent-ruby", "~> 1.0"
   spec.add_dependency "google-cloud-trace", "~> 0.33"
   spec.add_dependency "opencensus", "~> 0.5"
-  spec.add_dependency "google-cloud-monitoring", "~> 0.29.2"
+  spec.add_dependency "google-cloud-monitoring", "~> 0.30.0"
```
Comment (dazuma) [inline opencensus-stackdriver.gemspec]: Argh. Thanks for finding this. Please consider bumping to the latest version, `~> 0.35.0`.

Comment (igorpeshansky) [reply opencensus-stackdriver.gemspec]: I was trying to be conservative, as I couldn't find any good way to test this. If you believe that bumping to the latest carries the same amount of risk, I'm happy to do that.

Also, should we make this a bit less constrained and use `~> 0.35` instead?

Comment (dazuma) [reply opencensus-stackdriver.gemspec]: Oh shoot, you're right, there is an issue. 0.33 and later drop support for Ruby < 2.4. So in order to keep 2.2 and 2.3 support, we'd need to allow 0.32.

Okay, probably best would be `~> 0.32`.

Comment (igorpeshansky) [reply opencensus-stackdriver.gemspec]: Makes sense. Done.

**Review by dazuma — APPROVED** [review]

**Pull request merged**